htdocs/contact/document.php

```diff
@@ -168,7 +168,7 @@
 	$permission = $user->rights->societe->contact->creer;
 	$permtoedit = $user->rights->societe->contact->creer;
 	$param = '&id='.$object->id;
-	include DOL_DOCUMENT_ROOT.'/core/tpl/document_actions_post_headers.tpl.php';
+	include_once DOL_DOCUMENT_ROOT.'/core/tpl/document_actions_post_headers.tpl.php';
```

Comment:
Never use include_once on a .tpl file.   Goal of tpl is to include code inline.
If you use include_once, php may decide to not include the php code because it was already loaded.

Reply:
Hum.... No comment... I just added a PR that modifies all the files that include this tpl and that were all, I say all in include_once